### ISSUE TITLE
 diff is not listed in the report 

### ISSUE THREAD

**Issue by sag1v** [reg-viz/reg-cli#444]
For some reason (sometimes) when i run the visual tests (using [storycap](https://github.com/reg-viz/storycap) for capturing the images) reg-cli logs a certain diff (which is a valid change) but when i run the update script, it logs that 2 files were updated. The issue here is that the 2nd file was not even listed in the report.

When i push the changes to GitLab i can see a diff, it seems like the file size is the only thing that is different (i'm not sure why though).

It doesn't happen all the time so its hard to understand what is causing it, maybe storycap has a bug or a config i am missing that makes it generate a different image size.

Note that i am running the tests in a docker, so i doubt its an environment issue.

This is my `regconfig.json` file:

```
{
  "core": {
    "workingDir": ".reg",
    "actualDir": "somePath/__screenshots__",
    "thresholdRate": 0.3,
    "enableAntialias": true,
    "addIgnore": true,
    "ximgdiff": {
      "invocationType": "client"
    }
  }
}
``` 

and this is how i run it:

```
yarn reg-cli somePath somePath somePath -R somePath/report.html --extendedErrors --matchingThreshold 0.1
```

![image](https://user-images.githubusercontent.com/9153149/157621878-b2e36f54-4de2-4013-86fa-5f2101d3ed68.png)

**Comment by sag1v** [comment]
@bokuweb Do you have an idea what can cause this?

**Comment by Jeiwan** [comment]
You have this parameter passed:
`--matchingThreshold 0.1`
Which increases the tolerance. Unset it to see the diff.

**Comment by sag1v** [comment]
@Jeiwan Yeah i use it because sometimes i get random of false positive diffing.

**Comment by sag1v** [comment]
@bokuweb Do you have any idea what can cause this behavior?

reg-cli run and show 1 file changed
reg-cli with update actually updates 3 files
